src/components/LHNOptionsList/OptionRowLHN.tsx

```diff
@@ -131,33 +91,32 @@ function OptionRowLHN(props) {
             CONST.CONTEXT_MENU_TYPES.REPORT,
             event,
             '',
-            popoverAnchor,
-            props.reportID,
+            popoverAnchor.current,
+            reportID,
             '0',
-            props.reportID,
+            reportID,
             undefined,
             () => {},
             () => setIsContextMenuActive(false),
             false,
             false,
             optionItem.isPinned,
-            optionItem.isUnread,
+            optionItem.isUnread ?? undefined,
         );
     };
 
-    const emojiCode = lodashGet(optionItem, 'status.emojiCode', '');
-    const statusText = lodashGet(optionItem, 'status.text', '');
-    const statusClearAfterDate = lodashGet(optionItem, 'status.clearAfter', '');
+    const emojiCode = optionItem.status?.emojiCode ?? '';
+    const statusText = optionItem.status?.text ?? '';
+    const statusClearAfterDate = optionItem.status?.clearAfter ?? '';
     const formattedDate = DateUtils.getStatusUntilDate(statusClearAfterDate);
     const statusContent = formattedDate ? `${statusText ? `${statusText} ` : ''}(${formattedDate})` : statusText;
-    const isStatusVisible = !!emojiCode && ReportUtils.isOneOnOneChat(ReportUtils.getReport(optionItem.reportID));
+    const report = ReportUtils.getReport(optionItem.reportID ?? '');
+    const isStatusVisible = !!emojiCode && ReportUtils.isOneOnOneChat(isNotEmptyObject(report) ? report : null);
```

Comment:
Please use `!isEmptyObject` instead of isNotEmptyObject since it will be [deprecated](https://github.com/Expensify/App/issues/33886) soon